### DISCUSSION TITLE
glazing materials

### DISCUSSION THREAD

**Discussion by Gzhji** [mitsuba-renderer/mitsuba3#1095] [Q&A]
Dear community,

I was wondering how could we customize the color and transmittance for glazing materials?
Currently, Mitsuba can let us select dielectric materials.  
Lets say in the real world, there is a bluish window glass with 80% transmittance, is this possible to model it?

Thanks in advance!

Guanzhou

**Comment by njroussel** [comment]
Hi @Gzhji 

You might want to take a look at the [`principled` BSDF](https://mitsuba.readthedocs.io/en/latest/src/generated/plugins_bsdfs.html#the-principled-bsdf-principled). It has a fair bit of control over its appearance. However, the transmission/reflection ratio is still largely dictated by the Fresnel equations. If you want something fully custom, you of course [build your own BSDF](https://mitsuba.readthedocs.io/en/latest/src/others/custom_plugin.html).
